tests/test_gaudi_configuration.py

```diff
@@ -50,36 +26,6 @@ class GaudiConfigTester(unittest.TestCase):
     def test_default_parameter_types(self):
         gaudi_config = GaudiConfig()
 
-        self.assertIsInstance(gaudi_config.use_habana_mixed_precision, bool)
-        self.assertIsInstance(gaudi_config.hmp_is_verbose, bool)
         self.assertIsInstance(gaudi_config.use_fused_adam, bool)
         self.assertIsInstance(gaudi_config.use_fused_clip_norm, bool)
         self.assertIsInstance(gaudi_config.use_torch_autocast, bool)
-
-        self.assertTrue(is_list_of_strings(gaudi_config.hmp_bf16_ops))
-        self.assertTrue(is_list_of_strings(gaudi_config.hmp_fp32_ops))
-
-    def test_write_bf16_fp32_ops_to_text_files(self):
-        gaudi_config = GaudiConfig()
-
-        with tempfile.NamedTemporaryFile() as bf16_file:
-            with tempfile.NamedTemporaryFile() as fp32_file:
-                gaudi_config.write_bf16_fp32_ops_to_text_files(
-                    bf16_file.name,
-                    fp32_file.name,
-                )
-
-                self.assertTrue(
-                    filecmp.cmp(
-                        bf16_file.name,
-                        BF16_OPS_REFERENCE_FILE,
-                        shallow=False,
-                    )
-                )
-                self.assertTrue(
-                    filecmp.cmp(
-                        fp32_file.name,
-                        FP32_OPS_REFERENCE_FILE,
-                        shallow=False,
-                    )
-                )
```

Comment:
We should keep this test as `write_bf16_fp32_ops_to_text_files` will still be used to specify custom op lists.
I propose to instantiate a Gaudi config such as:
```py
gaudi_config = GaudiConfig(
    autocast_bf16_ops=[
        "add",
        "addmm",
        "bmm",
        "div",
        "dropout",
        "gelu",
        "iadd",
        "linear",
        "layer_norm",
        "matmul",
        "mm",
        "rsub",
        "softmax",
        "truediv",
    ],
    autocast_fp32_ops=[
        "embedding",
        "nll_loss",
        "log_softmax",
    ],
)
```